src/patterns/validation/index.md.njk

```diff
@@ -39,6 +39,12 @@ Validation should refuse to accept:
 
 For example, if you’re asking for someone’s date of birth you should not accept ‘13’ in the month field.
 
+Remove or ignore any whitespace in users’ answers where it’s not needed. If an invisible character causes a validation error, it can be hard for users to know what's gone wrong and how to fix it. Consider removing:
+
++ spaces entered as part of numbers and codes, such as postcodes or card details
++ leading or trailing whitespace that a user may have been accidentally copied and pasted
```

Comment:
Part of me wonders whether we should not even mention 'leading or trailing' but assume that any part of the user input could contain extra whitespace which should be accepted (unless it introduces ambiguity). https://github.com/alphagov/govuk-design-system/issues/1450#issuecomment-894094939 is an example where the whitespace was in the middle, this could potentially happen with copy paste too if copying multiple things or Word is doing something weird.